AgendaUWP/AgendaUWP/ViewModels/MainPageViewModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AgendaUWP.Models;
+using Prism.Commands;
+using Prism.Windows.Mvvm;
+using Prism.Windows.Navigation;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AgendaUWP.ViewModels
+{
+    class MainPageViewModel : ViewModelBase
+    {
+        #region properties
+        private INavigationService navigationService;
+        #endregion
+
+        #region constructors
+        public MainPageViewModel(INavigationService navigationService)
+        {
+            this.navigationService = navigationService;
+            initilizeCommands();
+            Debug.WriteLine("teste");
```

Comment:
This WriteLine is still necessary? If not, please, remove it

Reply:
Ok, done!